core/monitor.c

```diff
@@ -1261,6 +1261,9 @@ end_and_emit_trace(dcontext_t *dcontext, fragment_t *cur_f)
         md->emitted_size -= local_exit_stub_size(dcontext, target, md->trace_flags);
     }
 
+    /* Fixup all indirect trace exits to comform ibl routine */
+    IF_AARCH64(md->emitted_size += fixup_indirect_trace_exit(dcontext, trace));
```

Comment:
Something is missing from this diff: this function does not exist.

Also, while I don't know precisely what it does, are you sure this shouldn't be done during incremental trace building as each exit is added, rather than at the end.